net-print/brother-mfc7820n-bin/brother-mfc7820n-bin-2.0.1-r1.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils rpm linux-info
+
+DESCRIPTION="Brother printer driver for MFC-7820N"
+
+HOMEPAGE="http://support.brother.com"
+
+MY_PR=${PR/r/}
+
+SRC_URI="http://download.brother.com/welcome/dlf006265/brmfc7820nlpr-${PV}-${MY_PR}.i386.rpm
+        http://download.brother.com/welcome/dlf006267/cupswrapperMFC7820N-${PV}-${MY_PR}.i386.rpm"
+
+
+LICENSE="brother-eula GPL-2"
+
+SLOT="0"
+
+KEYWORDS="amd64 x86"
+
+RESTRICT="mirror strip"
+
+DEPEND="net-print/cups"
+RDEPEND="${DEPEND}"
+
+S=${WORKDIR}
+
+pkg_setup() {
+	CONFIG_CHECK=""
+	if use amd64; then
+		CONFIG_CHECK="${CONFIG_CHECK} ~IA32_EMULATION"
+	fi
+
+	linux-info_pkg_setup
+}
+
+src_unpack() {
+	rpm_unpack ${A}
+}
+
+src_prepare() {
```

Comment:
The whole function can then be removed.

Reply:
The prepare function is only needed if there is a patch. So the whole src_prepare can be removed.

Reply:
Yeah, but there is a patch now. Moving the postinst stuff into install, the cupswrapper script needs a patch to install to the sandbox.